beta/package.json

```diff
@@ -37,6 +37,7 @@
     "parse-numeric-range": "^1.2.0",
     "react": "18.0.0-alpha-930c9e7ee-20211015",
     "react-collapsed": "3.1.0",
+    "react-device-detect": "^2.1.2",
```

Comment:
This feels like an excessive dependency. What is the essence of the thing we're trying to check? What is the minimal way we can check it?

Reply:
I've removed it and using a regex match against `navigator.userAgent` now